app/src/main/java/com/hedvig/app/ui/compose/composables/ProgressIndicator.kt

```diff
@@ -0,0 +1,20 @@
+package com.hedvig.app.ui.compose.composables
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun CenteredProgressIndicator() {
+    Column(
+        modifier = Modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
```

Comment:
Maybe a Box with .fillMaxSize and only Alignment.Center as alignment feels more natural for this use case?